solidity/random-beacon/deploy/09_deploy_random_beacon_chaosnet.ts

```diff
@@ -33,3 +33,7 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 export default func
 
```

Comment:
@pdyraga
Just wanted to point out that we don't transfer ownership of `RandomBeaconChasonet` to any governance.
After the deployment the contract will still be owned by the deployer.
This is different compared to the real `RandomBeacon` which is owned by a governance.